tools/one_step_veloxbackend.sh

```diff
@@ -102,5 +97,7 @@ cd $GLUTEN_DIR/cpp
              --enable_hbm=$ENABLE_HBM --enable_s3=$ENABLE_S3 --enable_hdfs=$ENABLE_HDFS
 
 cd $GLUTEN_DIR
-mvn clean package -Pbackends-velox -Pspark-3.2 -Pspark-3.3 -DskipTests
+mvn clean package -Pbackends-velox -Pspark-3.2 -DskipTests
+mvn clean package -Pbackends-velox -Pspark-3.3 -DskipTests
+
```

Comment:
I agree to separate the building for spark 3.2 & 3.3. In addition, I note there is a CI job to verify building spark 3.2 & 3.3 simultaneously. Can this job be removed? It is not that necessary.

Reply:
Sounds good. will remove that.

Reply:
In hudi, it supports building all the spark version at the same time, but not support build release version at the same time, so I think we can track it in CI